keadm/cmd/keadm/app/cmd/edge/reset_others.go

```diff
@@ -83,56 +76,45 @@ func NewOtherEdgeReset() *cobra.Command {
 				}
 			}
 
-			staticPodPath := ""
+			step.Printf("clean up static pods")
 			config, err := util.ParseEdgecoreConfig(common.EdgecoreConfigPath)
 			if err != nil {
-				fmt.Printf("failed to get edgecore's config with err:%v\n", err)
+				klog.Warningf("failed to parse edgecore configuration, skip cleaning up static pods, err: %v", err)
 			} else {
 				if reset.Endpoint == "" {
 					reset.Endpoint = config.Modules.Edged.TailoredKubeletConfig.ContainerRuntimeEndpoint
 				}
-				staticPodPath = config.Modules.Edged.TailoredKubeletConfig.StaticPodPath
-			}
-			// first cleanup edge node static pod directory to stop static and mirror pod
-			if staticPodPath != "" {
-				if err := phases.CleanDir(staticPodPath); err != nil {
-					fmt.Printf("Failed to delete static pod directory %s: %v\n", staticPodPath, err)
-				} else {
-					time.Sleep(1 * time.Second)
-					fmt.Printf("Static pod directory has been removed!\n")
+				staticPodPath := config.Modules.Edged.TailoredKubeletConfig.StaticPodPath
+				if staticPodPath != "" {
+					if err := phases.CleanDir(staticPodPath); err != nil {
+						klog.Warningf("failed to delete static pod directory %s: %v", staticPodPath, err)
+					}
 				}
 			}
-
-			// 1. kill edgecore process.
-			// For edgecore, don't delete node from K8S
+			step.Printf("kill edgecore and remove edgecore service")
```

Comment:
Does kill EdgeCore need to wait to ensure that static pods are removed?

Reply:
If this refers to the cleanup of mirror pods generated by static pods, this work should be taken care of by edgecore. If there are any issues now, we can create a new issue to track.

Reply:
However, I added the process of waiting for edgecore to stop.